src/hip/hip_build_utils.cpp

```diff
@@ -140,6 +146,31 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
         return hsaco->path();
     }
     else
+#endif
+#ifdef MIOPEN_OFFLOADBUNDLER_BIN
```

Comment:
I think that both `MIOPEN_OFFLOADBUNDLER_BIN` and `EXTRACTKERNEL_BIN` are always defined, even if cmake is unable to find one or both binaries (the value of these macros is empty, but macros are still defined). Therefore I recommend removing these `#ifdefs`.

The selection can be (and is) performed at run time, basing on results of `isHccCompiler()` and `IsClangXXCompiler()`, right?

Reply:
Oops, I was wrong, please ignore.